CHANGELOG.md

```diff
@@ -1,6 +1,13 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html)
 
+# [v1.3.0](unreleased)
+### Notes
+- This release supports for Hypervisor Manager for API 800,1000,1200 
```

Comment:
You dont need to specify the resource name here as well as below. Just mention it as API * for the resources listed below.

Reply:
Done